default.nix

```diff
@@ -17,7 +18,7 @@
 
 let
   systemTable = {
-    x86_64-windows = builtins.currentSystem;
+    x86_64-windows = "x86_64-linux"; # builtins.currentSystem
```

Comment:
```suggestion
    x86_64-windows = "x86_64-linux"; 
```
Because windows is built similarly to the other review using wine?

Reply:
Yes!

Reply:
And it can only be built on `x86_64-linux` so far